old-opera-notes.html

```diff
@@ -4,8 +4,14 @@
 <meta charset="utf-8">
 <title>Notes</title>
 <style type="text/css">
+@media (prefers-color-scheme: dark) {  
+  .themed {  
+     background-color: #202124;    
+     color: #a5a5a5;  
+  }
+}
 body {font-size: 100%; margin: 1em auto; padding: 8px; width: 95%}
-select { width: 90%; margin: 1em auto; min-height:12em; display: block; resize: vertical; font-size: 1em}
+select { width: 90%; margin: 1em auto; min-height:12em; display: block; resize: vertical; font-size: 1em; overflow: auto}

```

Comment:
Scrollbar in the note selection box will be hidden if the amount of notes does not warrant it.